auth/server/src/main/java/com/seagates3/dao/ldap/AccountImpl.java

```diff
@@ -338,10 +338,66 @@ public void save(Account account) throws DataAccessException {
             throw new DataAccessException("failed to add new account.\n" + ex);
         }
 
-        createUserOU(account.getName());
-        createRoleOU(account.getName());
-        createGroupsOU(account.getName());
-        createPolicyOU(account.getName());
+        sleep();
+
+        int count = 0;
+        int maxTries = 2;
+        while (true) {
+          try {
+            createUserOU(account.getName());
```

Comment:
retry logic can be moved inside createUserOU

Reply:
Done